SW-Content-Register.txt

```diff
@@ -1,40 +1,10 @@
-NXP Product: NXP’s product with a part number of MIMXRT117HDVMAA or NXP’s product included in the NXP evaluation kit with a part number of SLN-VIZN3D-IOT
-
-SDK_Peripheral_Driver      Name: SDK Peripheral Driver                       
-                           Version: 2.x.x                                    
-                           Outgoing License: BSD-3-Clause                    
-                           License File: COPYING-BSD-3                       
-                           Format: source code                               
-                           Description: Peripheral drivers are designed for  
-                           the most common use cases identified for the      
-                           underlying hardware block.                        
-                           Location: devices/<device>/drivers                
-                           Origin: NXP (BSD-3-Clause)                        
-
-SDK_Examples               Name: SDK examples                                
-                           Version: NA                                       
-                           Outgoing License: BSD-3-Clause                    
-                           License File: COPYING-BSD-3                       
-                           Format: source code, binary, project files, linker
-                           files                                             
-                           Description: SDK out of box examples to show how  
-                           to use peripheral drivers and integrate           
-                           middleware.                                       
-                           Location: boards/<board>/                         
-                           Origin: NXP (BSD-3-Clause)                        
-
-SDK_Device                 Name: SDK SoC files                               
-                           Version: NA                                       
-                           Outgoing License: BSD-3-Clause                    
-                           License File: COPYING-BSD-3                       
-                           Format: source code, linker files                 
-                           Description: Device system files, template files, 
-                           IDE related startup and linker files.             
-                           Location: devices/<device>/                       
-                           Origin: NXP (BSD-3-Clause)                        
+Release Name: SLN-VIZN3D-IOT Software Development Kit
+Release Version: 1.4.0
+Outgoing license: LA_OPT_NXP_Software_License v49 September 2023 - Additional Distribution use, section 2.3 applies
+License file: LICENSE.txt
+Release location : https://github.com/NXP/vizn3d_smartlock_oobe
+Description: NXP’s product with a part number of MIMXRT117HDVMAA or NXP’s product included in the NXP evaluation kit with a part number of SLN-VIZN3D-IOT
```

Comment:
This line is not applicable anymore, the description should be different like: this release does that and that